nmdc_server/config.py

```diff
@@ -44,6 +44,7 @@ class Settings(BaseSettings):
     # for zipstreamer
     zip_streamer_url: str = "http://zipstreamer:4008/download"
     zip_streamer_chunk_size_bytes: int = 2 * 1024 * 1024
+    zip_streamer_nersc_data_host: str = "https://data.microbiomedata.org/data"
```

Comment:
I suggest calling it a "base URL" since it includes more than the host portion.

```suggestion
    zip_streamer_nersc_data_base_url: str = "https://data.microbiomedata.org/data"
```

Note: I am making this suggestion without making the corresponding suggestion in the function that consumes this environment variable.

Reply:
Works for me. I've changed it in both places with f777517